sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-data-inventory-report.sql

```diff
@@ -0,0 +1,505 @@
+                            sims.student_restrictions active_student_restrictions
+                        where
+                            active_student_restrictions.student_id = students.id
+                            and active_student_restrictions.is_active = true
+                    )
+            ) as "Active Restrictions Flag",
+            (
+                select
+                    ''Y''
+                where
+                    exists (
+                        select
+                            1
+                        from
+                            sims.student_restrictions inactive_student_restrictions
+                        where
+                            inactive_student_restrictions.student_id = students.id
+                            and inactive_student_restrictions.is_active <> true
+                    )
+            ) as "Previous (inactive) Restrictions Flag",
+            (
+                select
+                    SUM (disbursement_overawards.overaward_value)
+                from
+                    sims.disbursement_overawards disbursement_overawards
+                where
+                    disbursement_overawards.student_id = students.id
+                    and disbursement_overawards.disbursement_value_code = ''CSLF''
+                GROUP by
+                    disbursement_overawards.student_id,
+                    disbursement_overawards.disbursement_value_code
+            ) as "CSL Overawards Balance (FT)",
+            (
+                select
+                    SUM (disbursement_overawards.overaward_value)
+                from
+                    sims.disbursement_overawards disbursement_overawards
+                where
+                    disbursement_overawards.student_id = students.id
+                    and disbursement_overawards.disbursement_value_code = ''CSLP''
+                GROUP by
+                    disbursement_overawards.student_id,
+                    disbursement_overawards.disbursement_value_code
+            ) as "CSL Overawards Balance (PT)",
+            (
+                select
+                    SUM (disbursement_overawards.overaward_value)
+                from
+                    sims.disbursement_overawards disbursement_overawards
+                where
+                    disbursement_overawards.student_id = students.id
+                    and disbursement_overawards.disbursement_value_code = ''BCSL''
+                GROUP by
+                    disbursement_overawards.student_id,
+                    disbursement_overawards.disbursement_value_code
+            ) as "BCSL Overawards Balance",
+            students.disability_status as "Permanent Disability status",
+            -- Application Personal Info (does not include Institution or Program info)
+            ''"'' || program_years.program_year_desc || ''"'' as "Program Year",
+            applications.data ->> ''studentNumber'' as "Student Number",
```

Comment:
In the output csv file attached to the PR student number is blank in few places and null in few . 

![image](https://github.com/bcgov/SIMS/assets/54600590/e65ba0b2-4c0f-4ed4-8b81-fe9a69c67e5c)

Reply:
That's how it is in my local DB. I don't think it is an issue.

Reply:
Got it considering the report to be raw it should fine.